src/Main.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import "./Main.css"
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "../node_modules/bootstrap/dist/js/bootstrap.min.js"
+import "../node_modules/jquery/src/jquery.js"
+import "../node_modules/popper.js/dist/popper.min.js"
+
+
+class NavbarOption extends React.Component {
+  render() {
+    return <a href={this.props.link}>{this.props.text}</a>;
+  }
+}
+
+class Navbar extends React.Component {
+  renderNavbarOption(option, link) {
+    return <NavbarOption text={option} link={link} />;
+  }
+
+  render() {
+    return (
+   <nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+  <ul class="navbar-nav">
+    <li class="nav-item active">
+      <a class="nav-link" href="#">CollegePrep.org</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Link</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Link</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Link</a>
+    </li>
+  </ul>
+</nav>
+    );
+  }
+}
+
+// separation of selection box --> avatar, person's name, testimony
+
+/* Avatar */
+class Avatar extends React.Component {
+  render () {
+    return (
+      <div className="circle"><img className="avatar" src={this.props.img} /></div>
+      )
+  }
+}
+
+/* Person's name */
+class PersonInfo extends React.Component {
+  render () {
+    return (
+      <div className="bold-item">-{this.props.name}, {this.props.grade}</div>
+      )
+  }
+}
+
+/* Text */
+class Textbox extends React.Component {
+  render () {
+    return (
+      <div>{this.props.text}</div>
+      )
+  }
+}
+
+/* Testimony */
+class Testimony extends React.Component {
+  render () {
+    return (
+      <div className="col">
+        <div className="col1">
+          <Avatar img={this.props.img} />
+          <PersonInfo name={this.props.name} grade={this.props.grade} />
+        </div>
+        <div className="col2">
+          <Textbox text={this.props.content} />
+        </div>
+      </div>
+      
+      )
+  } 
+}
+
+
+/* Dropdown options */
```

Comment:
If we wanted to make this a reusable component, we shouldn't hardcode the dropdown options or button text here. It's probably ok to copy this code wherever we need a dropdown - this is generic enough / isn't worth the effort passing down an array of dropdown options along with their hrefs.